### PR TITLE
Add play/pause

### PR DESCRIPTION
**Pull request by EkT1oN**
- add new field Playstatus in DiscoBot{} struct
- add commands: 
    - disco-play 
    - disco-pause

### CODE REVIEW

**Comment by AlexSergEdge** [comment]
Greate code! Deploy to production immediately!

File: discobot.go [hunk 1]
```diff
@@ -87,7 +89,9 @@ func (bot *DiscoBot) playSound(s *discordgo.Session, guildID, channelID, url str
 		if packet.Timecode == webm.BadTC {
 			r.Shutdown()
 		} else {
-			vc.OpusSend <- packet.Data
+			if bot.Playstatus {
+				vc.OpusSend <- packet.Data
```
Comment (oleggator) [inline discobot.go:94]: In this case, stream chunks will be skipped. You have to wait here until the user will call `play` command.
1. Create `startPlayback chan struct{}` in `DiscoBot` struct
2. In the case when `PlayStatus=true` try to read from `startPlayback` channel.
3. In the `play` command handler write to `startPlayback` channel to continue playing.

File: discobot.go [hunk 2]
```diff
@@ -8,14 +8,16 @@ import (
...
 	"github.com/bwmarrin/discordgo"
 	"github.com/ebml-go/webm"
 	"github.com/kkdai/youtube/v2"
 	"github.com/schollz/progressbar/v3"
 )
 
 type DiscoBot struct {
-	api *discordgo.Session
+	api        *discordgo.Session
+	Playstatus bool
```
Comment (oleggator) [inline discobot.go]: Consider to rename to `PlayStatus`

Comment (EkT1oN) [reply discobot.go]: **done**

File: discobot.go (hunk 1 above)
Comment (EkT1oN) [reply discobot.go:94]: **done**

**Pull request merged**